src/styles/blocks/nav.scss

```diff
@@ -0,0 +1,45 @@
+.nav {
+  &__list {
+    display: flex;
+    padding: 0;
+    margin: 0;
+    list-style: none;
```

Comment:
in order not to write styles like `margin: 0; padding: 0;` for your list or `text-decoration: none;` for a tag you can add the these styles to your normalize.scss file and don't write the same reset styles each time

for example
```
ul {
  margin: 0;
  padding: 0;
  list-style-type: none;
}

a {
  text-decoration: none;
}
```

after you add these styles to your file don't forget do delete unnecessary styles from your scss blocks